src/Microsoft.DotNet.XHarness.CLI/Commands/WASM/WasmTestCommand.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading.Tasks;
+using Microsoft.DotNet.XHarness.CLI.CommandArguments;
+using Microsoft.DotNet.XHarness.CLI.CommandArguments.Wasm;
+using Microsoft.DotNet.XHarness.Common.CLI;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.DotNet.XHarness.CLI.Commands.Wasm
+{
+    internal class WasmTestCommand : TestCommand
```

Comment:
@MaximLipnin please merge the newest master and create a constructor that will pass the help message to the base ctor:

```csharp
        public WasmTestCommand() : base("Executes BCL xunit tests on WASM")
        {
        }
```

Reply:
Or ideally create a `const` and refernce it here and in `CommandDescription`

Reply:
added